frontend/src/Course.js

```diff
@@ -123,18 +153,52 @@ class Course extends React.Component {
             control={<Switch checked={this.props.isRequired} onChange={this.handleIsRequiredChange} />}
             label="Required"
           />
-          {this.props.sections.map((section, index) => (
-            <Section
+          <p>Lecture Sections</p>
+          {this.props.lectureSections.map((lectureSection, index) => (
+            <LectureSection
+              id={index}
+              key={index}
+              lectureSection={lectureSection}
+              professor={this.props.lectureSections[index].professor}
+              startTime={this.props.lectureSections[index].startTime}
+              endTime={this.props.lectureSections[index].endTime}
+              sun={this.props.lectureSections[index].days.indexOf(0) !== -1}
+              mon={this.props.lectureSections[index].days.indexOf(1) !== -1}
+              tue={this.props.lectureSections[index].days.indexOf(2) !== -1}
+              wed={this.props.lectureSections[index].days.indexOf(3) !== -1}
+              thur={this.props.lectureSections[index].days.indexOf(4) !== -1}
+              fri={this.props.lectureSections[index].days.indexOf(5) !== -1}
+              sat={this.props.lectureSections[index].days.indexOf(6) !== -1}
```

Comment:
Consider storing `days` in a set instead of an array since `indexOf` is an O(n) operation

Reply:
Right, I will make that change, thanks!